eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/MessageCleaner.java

```diff
@@ -0,0 +1,94 @@
+package io.eventuate.local.common;
+
+import io.eventuate.common.jdbc.EventuateSchema;
+import io.eventuate.common.jdbc.sqldialect.EventuateSqlDialect;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import javax.sql.DataSource;
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class MessageCleaner {
+
+  private EventuateSqlDialect eventuateSqlDialect;
+  private DataSource dataSource;
+  private EventuateSchema eventuateSchema;
+  private Boolean purgeMessagesEnabled;
```

Comment:
Why is this Boolean? Can it be nullable?

Reply:
Somehow I missed it, it should be boolean. I will review everywhere.